tests/integration/local/invoke/test_integrations_cli.py

```diff
@@ -718,7 +718,10 @@ def tearDown(self):
         for image in samcli_images:
             docker_client.images.remove(image.id)
 
-        shutil.rmtree(str(self.layer_cache))
+        try:
+            shutil.rmtree(str(self.layer_cache))
+        except:
+            pass
```

Comment:
You can also use, so that we won't need empty try/except block
```suggestion
        shutil.rmtree(str(self.layer_cache), ignore_errors=True)
```